src/components/Community/UserCard.tsx

```diff
@@ -39,12 +39,14 @@ const UserCardInner: FC<UserCardProps> = ({ name, terms, id }) => {
   });
 
   const data = userData?.usersById[0] as User;
-  const groups = data?.memberof?.communities
-    ?.flatMap(c => {
-      if (c) return c.groups?.map(g => g.name.toLowerCase());
-      return undefined;
-    })
-    .filter((x): x is string => x !== undefined);
+  const groups = [] as string[];
+  // TODO [ATS]: Resolve this
```

Comment:
Is that resolved?

Reply:
No. There should be an issue about the group management changes as we discussed yesterday.